source/index-management/index-management.rst

```diff
@@ -6,7 +6,7 @@ Index Management
 ================
 
 :Status: Accepted
-:Minimum Server Version: 2.4
+:Minimum Server Version: 3.6
```

Comment:
Bumping the min server version since 3.6 is our minimum supported server version and because index enumeration specification for server versions < 2.8 is removed in this PR.